src/main/scala/zio/zmx/MetricsDataModel.scala

```diff
@@ -103,10 +103,10 @@ trait MetricsDataModel {
     sealed case class Timer(name: String, value: Double, sampleRate: Double, tags: Chunk[Label]) extends Metric[Double]
 
     case object Zero extends Metric[Int] {
-      def name      = ""
-      def tags      = Chunk.empty
-      def value     = 0
-      def isEmpty() = true
+      val name    = ""
```

Comment:
why here you prefer vals? I have some expectations here will be races in init and some of vals can be null, but maybe i'm wrong 
+ why not just

```suggestion
    val Zero = new Metric[Int] {
      def name      = ""
      def tags      = Chunk.empty
      def value     = 0
    }
```

Reply:
@alesavin , I prefer to keep it val as there is no need for construction of each of the attributes for Metric using methods.